src/Logic.hs

```diff
@@ -535,7 +563,7 @@ tryIntegratePullRequest pr state =
       [ format "Merge #{}: {}" (prNumber, title)
       , ""
       , format "Approved-by: {}" [approvedBy]
-      , format "Auto-deploy: {}" [if approvalType == MergeAndDeploy then "true" else "false" :: Text]
+      , format "Auto-deploy: {}" [if approvalType == MergeAndDeploy || approvalType == MergeAndDeployOnFriday then "true" else "false" :: Text]
```

Comment:
There is a function `needsDeploy` in `Project.hs` that provides this information.